--- lpfo.c	2019-04-09 12:48:46.000000000 +0530
+++ lfp.c	2019-04-09 12:49:27.000000000 +0530
@@ -817,9 +817,9 @@
     const char *com_path = com_path_default;

     /* configuration file related */
-    char *global_cfg_path = "global_conf.json"; /* contain global (typ. network-wide) configuration */
-    char *local_cfg_path = "local_conf.json"; /* contain node specific configuration, overwrite global parameters for parameters that are defined in both */
-    char *debug_cfg_path = "debug_conf.json"; /* if present, all other configuration files are ignored */
+    char *global_cfg_path = "/etc/lora/global_conf.json"; /* contain global (typ. network-wide) configuration */
+    char *local_cfg_path = "/etc/lora/local_conf.json"; /* contain node specific configuration, overwrite global parameters for parameters that are defined in both */
+    char *debug_cfg_path = "/etc/lora/debug_conf.json"; /* if present, all other configuration files are ignored */

     /* threads */
     pthread_t thrid_up;